open_event/templates/gentelella/admin/event/wizard/step-1.html

```diff
@@ -182,8 +183,9 @@
         <div class="item form-group">
             <label>Ticket URL
             </label>
-            <input name="ticket_url" class="form-control"
-                   value="{{ event.ticket_url | default('', true) }}"/>
+            <input type="url" pattern="https?://.+" name="ticket_url" class="form-control"
```

Comment:
rather than using a pattern, why not use type=url as you have done above (social link) .